sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -17,20 +17,22 @@
 
 package org.apache.spark.sql.execution.datasources.jdbc
```

Comment:
Up to my knowledge, this is not the public API (see [execution/package.scala#L21-L23](https://github.com/apache/spark/blob/b85e29437d570118f5980a1d6ba56c1f06a3dfd1/sql/core/src/main/scala/org/apache/spark/sql/execution/package.scala#L21-L23)).